spring-cloud-skipper-docs/src/main/asciidoc/three-hour-tour.adoc

```diff
@@ -333,6 +357,107 @@ skipper:>search
 ╚═════════════════╧═══════╧════════════════════════════════════════════════════════════════════════════════╝
 ----
 
+[[create-our-package]]
+=== Creating Your Own Package
+
+For this package, we are going to be creating a super simple package, using the `helloworld` app above and uploading it to our local machine.
```

Comment:
We can use a different name than `helloworld` as it is already part of the out of the box package from `http://skipper-repository.cfapps.io/repository/experimental`. Will fix it.

Reply:
:+1: Thanks, sorry wasn't thinking!